openshift/rh-che.config.yaml

```diff
@@ -34,6 +34,7 @@ data:
   che.predefined.stacks.reload_on_start: "true"
   che-secure-external-urls: "true"
   che-server-java-opts: "-XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=25 -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -Xms50m -Dfile.encoding=UTF8"
+  che-service-account-name: "che-workspace"
```

Comment:
I believe we need a HK issue for adding new entry to cm, otherwise applying template on osd would fail since there would be no corresponding entry in the `rhche` cm on prod / prod-preview

Reply:
Ah I see. In that case is it better to include it as a value directly in the app yaml, or open a HK issue?